bookkeeper-proto/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -101,6 +101,7 @@ message ReadRequest {
     optional int64 previousLAC = 4;
     // Used as a timeout (in milliseconds) for the long polling request
     optional int64 timeOut = 5;
+    optional uint32 priority = 6 [default = 0];
```

Comment:
Can you make priority as part of PacketHeader? so this would be available to all type of requests.

As what I kind see here, it can be reused by `GetBookieInfoRequest`, `WriteLacRequest` and `ReadLacRequest` potentially.

Reply:
sure